app/views/devise/registrations/new.html.erb

```diff
@@ -1,34 +1,41 @@
-<h2>Create an account</h2>
+<div class="row">
+  <div class="small-6 columns small-centered">
+    <h2>Create an account</h2>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-  <%= devise_error_messages! %>
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= devise_error_messages! %>
 
-  <div class="field">
-    <%= f.label :username %><br />
-    <%= f.text_field :username, autofocus: true %>
-  </div>
+    <div class="field">
+      <!-- <%= f.label :username %><br /> -->
+      <%= f.text_field :username, autofocus: true, placeholder: 'Username' %>
+    </div>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: false %>
-  </div>
+    <div class="field">
+      <!-- <%= f.label :email %><br /> -->
+      <%= f.email_field :email, autofocus: false, placeholder: 'Email' %>
+    </div>
 
-  <div class="field">
-    <%= f.label :password %>
-    <% if @minimum_password_length %>
-    <em>(<%= @minimum_password_length %> characters minimum)</em>
-    <% end %><br />
-    <%= f.password_field :password, autocomplete: "off" %>
-  </div>
+    <div class="field">
+      <!-- <%= f.label :password, class: 'left' %> -->
+      <!-- <% if @minimum_password_length %>
+      <p class='right' id='passwordHelpText'>(<%= @minimum_password_length %> characters minimum)</p>
+      <% end %><br /> -->
+      <%= f.password_field :password,
+        autocomplete: "off",
+        placeholder: 'Password',
+        'aria-describedby' => 'passwordHelpText'
+      %>
+    </div>
 
-  <div class="field">
-    <%= f.label :password_confirmation %><br />
-    <%= f.password_field :password_confirmation, autocomplete: "off" %>
-  </div>
+    <div class="field">
+      <!-- <%= f.label :password_confirmation %><br /> -->
+      <%= f.password_field :password_confirmation, autocomplete: "off", placeholder: 'Confirm Password' %>
+    </div>
 
-  <div class="actions">
-    <%= f.submit "Sign up" %>
+    <div class="actions">
+      <%= f.submit "Sign up", class: 'button' %>
+      <%= render "devise/shared/links" %>
+    </div>
+    <% end %>
   </div>
-<% end %>
-
-<%= render "devise/shared/links" %>
+</div>
```

Comment:
Why are you commenting out portions of this form, as opposed to simply deleting them?